src/ci_providers/provider_local.ts

```diff
@@ -17,9 +17,10 @@ function _getBuildURL(): string {
 }
 
 function _getBranch(inputs: UploaderInputs): string {
-  const { args } = inputs
-  if (args.branch) {
-    return args.branch
+  const { args, environment: envs } = inputs
+  const branch = args.branch || envs.GIT_BRANCH || envs.BRANCH_NAME || ''
+  if (branch !== '' && branch !== undefined) {
```

Comment:
```suggestion
  if (branch !== '') {
```

The second condition will never be true. If any one of the sources are `undefined`, the final operand of the OR operator will be used.

Reply:
Good point...